src/ai/constants.rs

```diff
@@ -0,0 +1,25 @@
+pub const MODEL: &str = "gpt-3.5-turbo";
+pub const MODERATION_MODEL: &str = "text-embedding-ada-002";
```

Comment:
Should this be `EMBEDDING_MODEL`? We don't set the moderation model, automatically taking the latest one - this constant is used in embedding.